src/UI/Implementation/Component/ComponentHelper.php

```diff
@@ -9,6 +9,23 @@
  */
 trait ComponentHelper {
 	/**
+	 * Default implementation uses the namespace of the component up to and excluding
+	 * "Component", reverses the order and adds spaces.
+	 *
+	 * @return string
+	 */
+	public function getName() {
+		$cls = explode("\\", get_class($this));
+		$name = [];
+		$cur = array_pop($cls);
+		while ($cur !== "Component" && count($cls) > 0) {
+			$name[] = preg_replace("%([a-z])([A-Z])%", "$1 $2", $cur);
+			$cur = array_pop($cls);
+		}
```

Comment:
Hi @wanze, thx for the review. My opinion is not very strong on this. We could reuse this method in the KS-documentation, then I would slightly oppose, but in fact the method isn't used there now. What's @Amstutz saying?